ome-demoserver.yml

```diff
@@ -114,16 +114,16 @@
     - role: openmicroscopy.postgresql
       no_log: true
       postgresql_users_databases:
-      - user: "{{ vault.omero_server_db_user }}"
-        password: "{{ vault.omero_server_dbpassword }}"
-        databases: ["{{ vault.omero_server_dbname }}"]
+      - user: "{{ secrets.omero_server_db_user }}"
+        password: "{{ secrets.omero_server_dbpassword }}"
+        databases: ["{{ secrets.omero_server_dbname }}"]
 
     - role: openmicroscopy.omero-server
       omero_server_release: 5.4.4
```

Comment:
New var required here, like `demo_server_release` with inline default 5.4.4.

To be overridden by host var for `pub-omero`